.gitattributes

```diff
@@ -13,4 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Convert to CRLF on Windows since tests expect System.lineSeparator
+* text=auto
```

Comment:
I would prefer we apply this only to the runtime2 examples directory if possible. I think those files are really the only ones where the new lines *must* match the preferred OS line separator, since that's what the generator outputs now. All other files should just follow the users `core.autocrlf` settings that they think works best with their preferred development tools/OS.

Reply:
Even better, let's stop comparing the old example files to the new example files and just update them unconditionally during "sbt compile" like we update all the other automatically generated files.  Then we can either apply this setting only to the examples directory or remove both this setting and the `replace("\n", System.lineSeparator)` calls from CodeGeneratorState's generate methods.  Either way will guarantee that updated example code files are checked in with LFs, not CRLFs.